specification/refarch.adoc

```diff
@@ -147,311 +189,315 @@ assigned to the TVM by the VMM.
-transitions back to the OS/VMM, the TSM-driver will save the trigger CSRs 
-and associated debug states, thus not leaking any information to non-TEE 
-workloads. TVM self-hosted debug may be enabled from TVM creation time or 
-may be explicitly opted-into during execution of the TVM. The TSM may 
-invoke the TSM-driver to set up a TVM-specific trigger CSR state (per the 
+In order to support probe-mode debugging of the TSM, the RoT must support
+an authorized debug of the platform. The authentication mechanism used for
+debug authorization is implementation-specific, but must support the
+security properties described in the Section 3.12 of the RISC-V Debug
+Support specification version 1.0.0-STABLE <<R6>>. The RoT may support
+multiple levels of debug authorization depending on access granted. For
+probe-based debugging of the hardware, the RoT performing debug
+authentication must ensure that separate attestation keys are used for TCB
+reporting when probe-debug is authorized vs when the platform is not under
+probe-debug mode. The probe-mode debug authorization process must invalidate
+sealed keys to disallow sealed data access when in probe-debug modes. Note that
+the external debug opt-in control for the hosting supervisor domain must be
+independent from the confidential supervisor domain.
+
+When a TVM is under self-hosted debugging - on a transition to TVM
+execution, the TSM-driver must set up the trigger CSRs for the TVM. For TVM
+debugging, the TSM-driver may inhibit M and S/HS modes in the triggers. On
+transitions back to the OS/VMM, the TSM-driver will save the trigger CSRs
+and associated debug states, thus not leaking any information to non-TEE
+workloads. TVM self-hosted debug may be enabled from TVM creation time or
+may be explicitly opted-into during execution of the TVM. The TSM may
+invoke the TSM-driver to set up a TVM-specific trigger CSR state (per the
 configuration of the TVM).
 
 *Performance Monitoring considerations*
 
-By default the TSM and all TVMs run with performance monitoring suppressed. 
-If a TVM runs in this default mode (opted out of performance monitoring), 
-on a transition to the TVM, the TSM-driver enforces this via inhibiting the 
-counters (using mcountinhibit). 
+By default the TSM and all TVMs run with performance monitoring suppressed.
+If a TVM runs in this default mode (opted out of performance monitoring),
+on a transition to the TVM, the TSM-driver enforces this via inhibiting the
+counters (using mcountinhibit).
 
-The TVM may opt-in to use performance monitoring either at initialization 
-or post-init.
+The TVM may opt-in to use performance monitoring either at initialization or
+post-initialization of the TVM.
 
-If the TVM has opted-in to performance monitoring, the TSM may invoke the 
+If the TVM has opted-in to performance monitoring, the TSM may invoke the
 SBI PMU extension (via TSM-driver) or Supervisor counter delegation extension to
```

Comment:
Just FYI: We have assigned names for this which can be used. "Sscdeleg/Ssccfg"

Reply:
added